optados/src/electronic.f90

```diff
@@ -405,7 +405,7 @@ subroutine elec_read_band_curvature
 
        if(on_root) close (unit=curvature_unit)
 
-       ! Convert all band curvatures to eV Ang
+       ! Convert all band curvatures to eV Ang^2
        band_curvature=band_curvature*bohr2ang*bohr2ang*H2eV
```

Comment:
Be very careful here. OptaDOS does a double conversion here bohr2ang twice because of an error in the units of the optical matrix elements coming out of CASTEP. Are you sure that the band curvature has the same error?

Reply:
I think it has same error. I compared the calculated band curvature from a near-free electron band calculated as a chain of Li atoms in CASTEP with a polynomial fitting of the same band.

After unit conversion, the difference in curvature with both methods is around 0.02, which I think it is more likely to be an issue of the polynomial fitting.